nxdrive/metrics/metrics.py

```diff
@@ -0,0 +1,61 @@
+from logging import getLogger
+from queue import Empty, Queue
+from time import monotonic_ns
+from typing import TYPE_CHECKING
+
+from ..constants import TIMEOUT
+from ..engine.workers import Worker
+from ..exceptions import ThreadInterrupt
+from ..objects import Metrics
+from ..options import Options
+from ..qt.imports import pyqtSlot
+from .constants import SYNC_ACTION, SYNC_TIME
+
+log = getLogger(__name__)
+
+_ENDPOINT = "/site/api/v1/me"
```

Comment:
That would be more logical to move it to `metrics.constants` and thus removing the "_" prefix.